bin/yugabyted

```diff
@@ -2283,5 +2487,151 @@ def retry_op(func, timeout=180):
+
+    # Create YCQL Keyspace
+    if isinstance(proxy_class, YcqlProxy) and is_valid(container) and \
+            not proxy_class.keyspace_exists(container):
+        Output.log("Creating {} keyspace".format(container))
+        _, err = proxy_class.create_keyspace(container)
+        if err:
+            Output.log_error_and_exit("Failed to create {} keyspace: \
+                {}".format(container, err))
+
+        Output.log("{} keyspace created successfully".format(container))
+
+    # Update password for default user
+    if is_valid(password) and not is_valid(user):
+        _, err = proxy_class.update_password(password)
+        if err:
+            Output.log_error_and_exit("class: {} Failed to update password for default user {}".
+                format(proxy_class, err))
+
+        Output.log("class: {} Password has updated for default user successfully".
+            format(proxy_class))
+
+    # Create user 
+    if is_valid(user) and not proxy_class.user_exists(user):
+        Output.log("Creating {} user".format(user))
+        _, err = proxy_class.create_user(user, password if is_valid(password) else user)
+        if err:
+            Output.log_error_and_exit("Failed to create {} user: {}".format(user, err))
+
+        if isinstance(proxy_class, YsqlProxy):
+            if not is_valid(container) and not proxy_class.db_exists(user):
+                Output.log("Creating {} database".format(user))
+                _, err = proxy_class.create_db(user)
+                if err:
+                    Output.log_error_and_exit("Failed to create {} database: {}".format(user, err))
+
+                Output.log("{} database created successfully".
+                    format(user))
+
+            _, err = proxy_class.db_owner(container if is_valid(container) else user, user)
+            if err:
+                Output.log_error_and_exit("Failed to change owner on {} database: {}".
+                    format(user, err))
+
+        if isinstance(proxy_class, YcqlProxy) and not is_valid(container) and \
+                not proxy_class.keyspace_exists(user):
+            Output.log("Creating {} keyspace".format(user))
+            _, err = proxy_class.create_keyspace(user)
+            if err:
+                Output.log_error_and_exit("Failed to create {} keyspace: {}".format(user, err))
+
+            Output.log("{} keyspace created successfully".format(user))
+
+        # Note: Following lines will be commented till we can decide on default user deletion.
+        #_, err = proxy_class.delete_user(user, password if is_valid(password) else user)
+        #if err:
+        #   Output.log_error_and_exit("Failed to delete default user: {}".format(err))
+        
+        Output.log("{} user created successfully".format(user))
+
+    Output.log("Custom credentials configured successfully")
+
+
+def get_env_credentials(yb_api_client_program, use_env_credentials=True):
```

Comment:
minor: the use_env_credentials=True parameter could be better named as get_default_credentials=False